app/controllers/mailings_controller.rb

```diff
@@ -1,17 +1,15 @@
 class MailingsController < ApplicationController
 
-  load_and_authorize_resource
+  load_and_authorize_resource except: :index
 
   def index
     @mailings = Mailing.order('id DESC').page(params[:page])
     authorize! :read, :mailing
   end
 
-  def show
-    @mailing = Mailing.find(params[:id])
-  end
-
+  # These actions are here to enable the cancancan 'not authorised' notice
```

Comment:
Thanx for the comment 🙏 
Will for sure help people looking at this in the future 🖤